portfolio-project-matching-app/models/FilteredProject.js

```diff
@@ -69,19 +138,6 @@ class FilteredProject {
         // search failed, return false
         return false;
     }
-
-    technologiesIsEmpty() {
-        /*
-        DESCRIPTION:    indicates whether project is associated with zero
-                        technologies
-
-        INPUT:          NA
-
-        RETURN:         boolean indicating whether project is associated with
-                        zero technologies
-        */
-        return this.project.technologies === null || this.project.technologies.length === 0;
-    }
```

Comment:
I had this method duplicated in both `FilteredProject` and `Project`. However, `FilteredProject` has a `Project`, so it was redundant. Went ahead and aligned with OOP best practices.